src/content/docs/alerts/get-notified/notification-integrations.mdx

```diff
@@ -738,6 +738,13 @@ Read more about each of our specific notification integrations.
          alt="A screenshot permissions configurations."
          src="/images/accounts_screenshot-crop_slack-destination-allow-access.webp"
        />
+       <Callout variant="important">
+         Two things happen when authenticating for the first time.
+         1. OAuth handshake which creates a token for authenticating with Slack
+         2. The user's personal token is saved (which allows creating a channel to a private slack channel)
+         This means that even if the original user is removed from the slack workspace, the destination continues to send notifications.
```

Comment:
I saw in the preview that my newline is not actually creating a new line. do i need to add \n? another tag?

Reply:
maybe this doesn't need to be a callout, its bigger than the image before it 🤔

Reply:
I'll take a look!